abci/client/grpc_client.go

```diff
@@ -35,6 +36,7 @@ func NewGRPCClient(addr string, mustConnect bool) Client {
 	cli := &grpcClient{
 		addr:        addr,
 		mustConnect: mustConnect,
+		chReqRes:    make(chan *ReqRes, 64),
```

Comment:
what if there will be more than 64 txs (responses)?

Reply:
Then the next async call will block until one of the pending callbacks have been processed (note callbacks, since the actual requests/responses are always processed serially). The main implication of this is that it sets an upper bound on the number of calls that can be involved in a deadlock - in this case, it's impossible to submit 65 async calls that have a cyclical dependency. It also allows up to 64 gRPC requests to be executed and responses queued while a slow callback at the head of the queue is being processed.

Reply:
thanks. can we add a comment

Reply:
Yup. Not exactly "small", but there's a comment now. :)